src/todolist.js

```diff
@@ -0,0 +1,49 @@
+export default class ToDolist {
+  #toDo;
+
+  constructor() {
+    this.#toDo = [];
+  }
+
+  SetToDo(toDo) {
+    this.#toDo = toDo;
+  }
+
+  get ToDo() {
+    return this.#toDo;
+  }
+
+  reorder() {
+    for (let index = 0; index < this.#toDo.length; index += 1) {
+      this.#toDo[index].id = index;
+    }
+  }
+
+  // Methods
+  AddToDo(activity, doneBox) {
+    const id = this.#toDo.length;
+    this.#toDo.push({
+      activity,
+      doneBox,
+      id,
```

Comment:
- [ ] Please asdemanded by the project requirements, the properties of each task should be `description`, `completed` and `index`. Kindly fix that please.